libraries/Wire/Wire.h

```diff
@@ -28,9 +28,7 @@
 
 #include "RingBuffer.h"
 
-#define BUFFER_LENGTH 32
```

Comment:
Sounds like it needs the nordic libraries, which this repository doesn't use. You'd probably get better support creating an issue in https://github.com/jrowberg/i2cdevlib